build.gradle

```diff
@@ -30,7 +30,7 @@ java {
 dependencies {
     compileOnly "org.embulk:embulk-api:0.10.31"
     compileOnly "org.embulk:embulk-spi:0.10.31"
-    compile "org.mongodb:mongo-java-driver:3.8.1"
+    compile "org.mongodb:mongo-java-driver:3.12.11"
```

Comment:
This is the latest version of v3 series.
https://mvnrepository.com/artifact/org.mongodb/mongo-java-driver